types/stubs/logging/__init__.pyi

```diff
@@ -0,0 +1,353 @@
+from typing import Any, Optional
```

Comment:
Yikes wonder if you can submit this as a patch somewhere!

Reply:
Yep!  Someone could submit a PR on typeshed.  I suspect the reason they haven't been added is that they're hoping to get some custom stuff in to handle the format strings and additional args, though.